internal/web_server/web/handlers/quiz_pages_handlers.go

```diff
@@ -110,3 +115,34 @@ func (qph *QuizPagesHandler) getArchivedQuizzes(c echo.Context) error {
 func (qph *QuizPagesHandler) getQuizProfile(c echo.Context) error {
 	return utils.Render(c, http.StatusOK, quiz_pages.QuizProfile())
 }
+
+func (qph *QuizPagesHandler) usernamePage(c echo.Context) error {
+
+	user, error := users.GetUserByID(qph.sharedData.DB, utils.GetUserIDFromCtx(c))
+	if error != nil {
+		return error
+	}
+
+	return utils.Render(c, http.StatusOK, quiz_pages.UsernamePage(user))
+}
+
+// Adds phone number and competition opt-in to user
```

Comment:
"competition opt-in to user": is it not more accurate to say "leaderboards opt-in status for user"

Reply:
både og, ekke det sånn at dei e nødt til å være med på leaderboardsa for å være med i konkuransen?

Reply:
Changed to "Adds phone number and the leaderboards opt-in status to the user"